plugin/evm/atomic/atomic_trie_iterator_test.go

```diff
@@ -20,7 +20,7 @@ import (
 func TestIteratorCanIterate(t *testing.T) {
 	lastAcceptedHeight := uint64(1000)
 	db := versiondb.New(memdb.New())
-	codec := TestTxCodec
+	codec := testTxCodec
```

Comment:
nit: should we inline this and replace all references to `codec` with `testTxCodec`?